macosx/English.lproj/MainMenu.xib

```diff
@@ -410,17 +410,17 @@
                 <menuItem title="Help" id="1431">
                     <menu key="submenu" title="Help" id="1429">
                         <items>
-                            <menuItem title="HandBrake User Guide" keyEquivalent="?" id="1985">
+                            <menuItem title="HandBrake Website" id="1432">
                                 <connections>
-                                    <action selector="openUserGuide:" target="6lr-Yy-GMc" id="Fda-3E-7Ya"/>
+                                    <action selector="openHomepage:" target="6lr-Yy-GMc" id="KTB-6B-jIe"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="HandBrake Homepage" id="1432">
+                            <menuItem title="HandBrake Documentation" keyEquivalent="?" id="1985">
```

Comment:
One thing I do is "HandBrake Documentation (HTTP)" to signal to uses that this will take them to a URL out the app.